apps/tangle-dapp/containers/DelegateTxContainer/DelegateTxContainer.tsx

```diff
@@ -49,6 +51,24 @@ const DelegateTxContainer: FC<DelegateTxContainerProps> = ({
     [getActiveValidators.name],
     ([, ...args]) => getActiveValidators(...args)
   );
+  const { data: waitingValidatorsData } = useSWR(
+    [getWaitingValidators.name],
+    ([, ...args]) => getWaitingValidators(...args)
+  );
+
+  const [maxNominationQuota, setMaxNominationQuota] = useState<number>(0);
+
+  useEffect(() => {
+    getMaxNominationQuota().then((maxNominationQuota) => {
+      setMaxNominationQuota(maxNominationQuota ? maxNominationQuota : 16);
+    });
+  }, []);
+
+  const allValidators = useMemo(() => {
+    if (!activeValidatorsData || !waitingValidatorsData) return [];
+
+    return [...activeValidatorsData, ...waitingValidatorsData];
```

Comment:
Is there any case where we have duplicate validators? Just out of curiosity.

Reply:
No there won't be.